### ISSUE TITLE
VSCode failing to launch external program npm in 1.3.1

### ISSUE THREAD

**Issue by aflanagan** [Microsoft/vscode#9710]
- VSCode Version: 1.3.1
- OS Version: OSX 10.11.1

I have the following task set up that I have been using for quite a while to run a few different npm scripts

```
// A task runner for npm scripts
{
  "command": "npm",
  "isShellCommand": true,
  "showOutput": "always",
  "args": ["run"],
  "isWatching": false,
  "tasks": [
    { "taskName": "build" },
    { "taskName": "watch"	},
    { "taskName": "watch:all"	}
  ]
}
```

this has worked well for me across several versions of vscode, but on Friday when I upgraded to 1.3.1 I started getting the following error when trying to start any of these tasks
```
Failed to launch external program npm run.
spawn npm ENOENT
```
Nothing has changed wrt to my npm version/installation path. Any suggestions for getting this working again?

**Comment by wallverb** [comment]
Try 1.4.0-insiders version https://github.com/Microsoft/vscode/issues/8434

**Comment by aflanagan** [comment]
Thanks, I can confirm it is fixed in 1.4.0-insiders.
